### ISSUE TITLE
When the model training is completed, what is the loss value?

### ISSUE THREAD

**Issue by bubbliiiing** [xuannianz/keras-CenterNet#38]
When the model training is completed, what is the loss value? 
The value of htloss, wh loss and reg loss.
My detection performance is very poor.  I want to see if there is a loss problem.
Already stared .
Thank u.  
: )
